docker/falco/Dockerfile

```diff
@@ -22,73 +22,36 @@ RUN apt-get update \
 	bash-completion \
 	bc \
 	bison \
-	clang-7 \
 	ca-certificates \
+	clang \
+	cpio \
 	curl \
 	dkms \
+	dwarves \
 	flex \
-	gnupg2 \
 	gcc \
+	gcc-11 \
+	gnupg2 \
+	gpg \
 	jq \
 	libc6-dev \
 	libelf-dev \
+	libiberty-dev \
+	libncurses-dev \
+	libpci-dev \
 	libssl-dev \
-	llvm-7 \
-	netcat \
-	patchelf \
+	libudev-dev \
+	llvm \
+	lsb-release \
+	netcat-openbsd \
+	openssl \
+	rpm2cpio \
+	software-properties-common \
+	wget \
 	xz-utils \
+	zstd \
```

Comment:
This list seems a bit too long :angel: 

Perhaps, some of them are not needed. Could we double check, please?

Reply:
Great catch! I'll check with `falco-driver-loader` to see what we actually need. :+1:

Reply:
Done! I tested falco-driver-loader with download and compilation of the kmod and ebpf probe.